.github/workflows/pr_on_all_plugins.yml

```diff
@@ -0,0 +1,209 @@
+          fi
+      - name: Open SDK Update Issue
+        id: open-issue
+        run: |
+          # Create a new issue with the SDK reference in the SDK repository
+          issue_url=$(gh issue create \
+                        --repo ${{ github.repository }} \
+                        --label auto \
+                        --title "Bumping Eth plugin SDK to latest version ${{ needs.get-sdk-ref.outputs.sdk_ref }}" \
+                        --body "Placeholder")
+          echo "Issue URL: $issue_url"
+          # Set output
+          echo "issue_url=$issue_url" >> $GITHUB_OUTPUT
+    outputs:
+      issue_url: ${{ steps.open-issue.outputs.issue_url }}
+
+
+  open-prs:
+    name: Open pull requests
+    runs-on: ubuntu-latest
+    needs: [build-repositories-matrix, open-issue, get-sdk-ref]
+    strategy:
+      fail-fast: false
+      matrix:
+        repo: ${{ fromJSON(needs.build-repositories-matrix.outputs.repo-matrix) }}
+    env:
+      GITHUB_TOKEN: ${{ secrets.CI_BOT_TOKEN }}
+
+    steps:
+      - name: Checkout plugin repository
+        # TODO: remove restriction once finalized
+        if: matrix.repo == 'app-plugin-uniswap-test2'
+        uses: actions/checkout@v3
+        with:
+          repository: LedgerHQ/${{ matrix.repo }}
+          submodules: recursive
+          ref: develop
+          # by default the action uses fetch-depth = 1, which creates
+          # shallow repositories from which we can't push
+          fetch-depth: 0
+          # needed, else the push inside the action will use default credentials
+          # instead of provided ones
+          persist-credentials: false
+
+      - name: Update submodule
+        # TODO: remove restriction once finalized
+        if: matrix.repo == 'app-plugin-uniswap-test2'
+        run: |
+          cd ethereum-plugin-sdk/
+          git checkout ${{ needs.get-sdk-ref.outputs.sdk_ref }}
+
+      - name: Commit changes
+        # TODO: remove restriction once finalized
+        if: matrix.repo == 'app-plugin-uniswap-test2'
+        id: commit-changes
+        run: |
+          # Set credentials for commit creation
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          # Branch name and title will be unique by design
+          branch_name="auto/bump_sdk_to_${{ needs.get-sdk-ref.outputs.sdk_ref }}"
+          title="[auto-update] Bump SDK to latest develop version ${{ needs.get-sdk-ref.outputs.sdk_ref }}"
+          echo "Branch Name: $branch_name"
+          echo "Title: $title"
+          git status
+          git commit -am "$title"
+          # Set output
+          echo "title=$title" >> $GITHUB_OUTPUT
+          echo "branch_name=$branch_name" >> $GITHUB_OUTPUT
+
+      - name: Push commit
+        # TODO: remove restriction once finalized
+        if: matrix.repo == 'app-plugin-uniswap-test2'
+        uses: ad-m/github-push-action@master
```

Comment:
I dont know wether it's best to use gh or a dedicated action here, but as long as it is working ... :)

Reply:
I tried using gh for a long time but gave up, too many issues with permissions / signer rights / etc